Components/Badge/badge.css

```diff
@@ -0,0 +1,63 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+
+.badge-conatiner {
+  display: flex;
+  flex-direction: row;
+}
+
+.badge {
+  position: relative;
+}
+
+.badge-icon {
+  position: absolute;
+  bottom: 60%;
+  right: 40%;
+  color: var(--text-color-inverted);
+  font-weight: bolder;
+  border-radius: 50%;
+  font-size: var(--copyrightText);
+  padding: 0 var(--space-xs);
+  background-color: var(--primary-sky-color);
+}
+
+/* in-stock colors */
+
+.colors {
+  display: flex;
+  position: relative;
+  display: inline;
+
+}
```

Comment:
Here you can either keep flex or inline, only the display: inline will work here as it comes later in the document. If you wanted to use flex but did not want create a new block then you can use 'display: inline-flex'.